### PR TITLE
Add my reservation component

### PR DESCRIPTION
**Pull request by imranhossainemi**
### Description

- [ ] Add my reservation page component with demo reservation list to project from add_homepage_component branch.

#### General

- [ ] The code follows the project's coding guidelines and best practices.
- [ ] The code is modular, well-organized, and easy to understand.
- [ ] Variable and function names are meaningful and self-descriptive.
- [ ] There are  commented-out code blocks or unused variables/methods for api declaration for future use

#### Functionality

- [ ] The changes accomplish the intended purpose and work as expected.
- [ ] The existing functionality is not affected adversely by the changes.
- [ ] All edge cases and possible error scenarios have been considered.

#### Testing

- [ ] The changes include appropriate unit tests for new functionality have not been tested.
- [ ] Test for responsive design

### CODE REVIEW

File: src/pages/ReservationPage.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchReservations, removeReservation } from '../redux/reservations/reservationSlice';
+import './ReservationPage.css';
```
Comment (shasherazi) [inline src/pages/ReservationPage.jsx:4]: this css route has a typo. it should be `./reservationPage.css` (first R is small case)

**Review by shasherazi — APPROVED** [review]
Hello @imranhossainemi, everything looks good. I have added a comment for a small typo in an import path, other than that it is ready to merge.

**Review by byiringiroscar — APPROVED** [review]
Hi @imranhossainemi ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
